docs/tables/oci_application_migration_migration.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "Steampipe Table: oci_application_migration_migration - Query OCI Application Migration Service Migrations using SQL"
+description: "Allows users to query data related to migrations in Oracle Cloud Infrastructure's Application Migration Service."
+---
+
+# Table: oci_application_migration_migration - Query OCI Application Migration Service Migrations using SQL
+
+**Deprecated. Use [azuread_group](https://hub.steampipe.io/plugins/turbot/azuread/tables/azuread_group) instead.**
```

Comment:
Please update the reference to use the corresponding OCI table and not the `azuread_*` table.

Should we keep the example queries?